src/modm/board/nucleo_f103rb/board.hpp

```diff
@@ -67,7 +67,10 @@ struct SystemClock {
 	{
 		Rcc::enableInternalClock();	// 8MHz
 		// internal clock / 2 * 16 = 64MHz, => 64/1.5 = 42.6 => bad for USB
```

Comment:
Is there a reason why we don't set the internal clock to 72MHz (and have 48MHz for USB)?

```c++
// internal clock / 2 * 18 = 72MHz, => 72MHz/1.5 = 48MHz for USB
const Rcc::PllFactors pllFactors{
  .pllMul = 18,
};
```

Reply:
Because the PLL can only do x16 and there is a fixed /2 prescaler from HSI: 8MHz/2*16 = 64MHz. If you had a 8MHz external clock, this could work.

Reply:
Besides, afaik the F103 doesn't have USB, that would be 105 or 107?